### PR TITLE
Refactor queen follow topics

### PR DESCRIPTION
**Pull request by psykzz**
## About The Pull Request

Cleans up a little duplicate code on queen following while ovi.
Changes `(watch)` -> `(F)` which people should be use to from observing. 

image

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/queen/queen.dm [hunk 1]
```diff
@@ -190,33 +190,20 @@
...
-		if(target == observed_xeno)
-			set_queen_overwatch(target, TRUE)
-		else
-			set_queen_overwatch(target)
-
-	if (href_list["watch_xeno_number"])
-		if(!check_state())
-			return
 		var/xeno_num = text2num(href_list["watch_xeno_number"])
 		for(var/mob/living/carbon/Xenomorph/X in hive.get_watchable_xenos())
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/xenomorph/castes/queen/queen.dm]:
```suggestion
		for(var/Y in hive.get_watchable_xenos())
			var/mob/living/carbon/Xenomorph/X = Y
```

File: code/modules/mob/living/carbon/xenomorph/hive_datum.dm [hunk 2]
```diff
@@ -81,6 +81,8 @@
 			var/mob/living/carbon/Xenomorph/X = i
 			if(is_centcom_level(X.z))
 				continue
+			if(X.stat == DEAD)
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/hive_datum.dm]: dead xenos arent in these lists.

**Comment by psykzz** [comment]
@spookydonut happy to TM again?

**Comment by Rohesie** [comment]
This has been tested already and the change does not affect functionality.

**Pull request merged**